data/remnant/remnant missions.txt

```diff
@@ -4479,3 +4479,140 @@ mission "Remnant: Expanded Horizons Astral job"
+			`	The display changes as an overlay of haze appears, pulsing in intensity. "Most interestingly, though, is that this area is significantly more variable than the rest of the galaxy. The haze shown here represents the heat hazard schematic we produce for our captains. It combines the local density with the measured heat values to estimate how much heat ships will be exposed to in the area."`
+			choice
+				`	"Interesting! Is that what causes the red glow?"`
+				`	"Wait a second. Why are there letters in your numbers?"`
+					goto numericsystem
+				`	"Thanks! I think that is all the details I can handle right now."`
+					goto stormend
+			label stormresume
+			`	"Not really, although it does contribute. Most of the red glow seems to be the result of clouds of hydrogen gases at aproximately 2710 degrees, rather like the process documented by early Earth astronomers investigating nebulae. Except for most of these clouds, there is no obvious source of energy to keep the hydrogen excited. Obviously there is still energy being added to the system, but..." She trails off and gestures depreciatingly. "One more mystery to keep us alert and attentive."`
+			`	She smiles thoughtfully, and the display changes with the red haze vanishing and a silvery haze replacing it. "But that is where today's research comes in. We have been measuring these storms since we arrived in the Ember Waste. The human eye can only pick out a few of them on the display, but our algorithms have identified a significant number of ripples of energy." She appears thoughtful.`
+			`	"It is almost like watching the surface of a pond during a rainstorm: The surface appears to be chaotic, but with detailed enough instrumentation, one can pick out individual ripples and trace their paths backwards... Except we have yet to find an actual source for any of them. We can trace every ripple on a pond in a storm and find its origin, but we have yet to learn the dance that we have lived in for half a millennia."`
+			choice
+				`	"What do the storms have to do with the heat?"`
+				`	"What units do you use for temperature?"`
+					goto tempunits
+				`	"I think that is all the technical detail I can take."`
+					goto stormend
+			`	She looks mischievous. "I can not tell you where or how we got it, but look at this." Her display shifts again with the hazes vanishing, and shows the galaxy map with one key difference. "Yes, the Ember Wastes are violet. Actually, they are off the visual spectrum into the ultraviolets, but this downshifts it so we can see it. This is what the Ember Wastes looked like a very long time ago.`
```

Comment:
Is it supposed to be some sort of convention to deliberately mis-spell 'cannot' as 'can not' in ES? I see it in a lot of places and it's really confusing. I understand default locale is en_US, but I wouldn't have thought this is a (real-world) regional variation.

Reply:
According to Merriam-Webster, both cannot and can not are valid words, and the definition of cannot is actually can not. According to them, both have the same definition, however the former (cannot) is much more common today than it is as separate words. It does note that if the word "not" is part of a phrase or saying, then it *must* be separate words. (example from grammarly: We can not only break even, but also turn a profit.)

https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference

Grammarly, which is focused a bit more on "make your writing look good today" and not so much on "what is technically correct," puts it a bit stronger in that one shouldn't use "can not" unless it is as part of the phrase structure described by Merriam Webster.

That being said, the fact that the Remnant never use "can't" is a deliberate stylization choice. Their language doesn't use those contractions.

Reply:
As a native English speaker (albeit from a British/Australian background) I understand the difference between when to use 'cannot' vs 'can not' as you mentioned. My point is that in this context it ought to be cannot to avoid that ambiguity. But after having a similar discussion with prose for The Battle for Wesnoth (where the species concerned similarly avoids contractions like can't) and your mention of Miriam-Webster, maybe it is a regional difference after all? Strangely, it is only in these two FOSS projects where I've seen American authors use 'can not' where I've normally seen it rendered as cannot, plus in my experience I've observed Americans usually combine separate words into single words so it is rather odd to see the reverse case here.

Ah well, just trying to help by pointing out errors (and the prose in ES has been mostly free of typographical errors that I've seen, at least so far).

Reply:
Sorry if I came across as implying you didn't know it. It's just that in the past, dealing with spelling/grammar issues go a lot smoother if we go directly to authoritative resources. 

In any case, I know that, personally, having others reading through stuff and pointing out oddities like this is helpful in getting me to go back and objectively read stuff again. I'm currently considering putting some of these, at least, up for consideration for changing.

Interesting facts: There are only six instances of `can not` in the Endless Sky datafiles (only counts stuff in vanilla, does not include un-merged PRs). In contrast, there are 109 `cannot` and 251 `can't`

Reply:
No worries, admittedly I wrote the original remark in haste just before I left for the afternoon/evening. I've just spent the last month (first-time) playing v0.9.12 (and having a lot of fun with it too), and have only just started on the v0.9.14 content, self-compiled now that I understand publishing on Steam/PPA currently isn't an option. Whenever I see (what looks to me like) a typo, it bugs me, so I track down the commit/PR and point it out. It's what I'm accustomed to, especially if it's only a word here or there (as opposed to a full-text review), and it notifies people involved in the original PR (at least in my e-mail notifications I get the full comment) - but if new issues are preferred in this project I can do that.

I had a look at those same six instances (at v0.9.14): only three are spoken by Remnant members in `remnant missions.txt`, the others are in `coalition.txt` (Heliarch speaker), `human/transport missions.txt` (spoken by Alondo), and `remnant news.txt` (narrator). None of them are of the 'can not-X' form. If only for consistency with existing convention, I'd suggest changing to 'cannot' where possible.

Reply:
I just made a PR to change these. I'll suggest that further discussion of this should happen there.

As a side note, kudos for digging back to where the word was introduced to track the history and why (if any) they were used that way. Most people aren't so thorough.